Runtime/Data/Attribute/RestrictedAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Zinnia.Data.Attribute
+{
+    using UnityEngine;
+    using System;
+
+    /// <summary>
+    /// Defines the <c>[Restricted]</c> attribute.
```

Comment:
```suggestion
    /// Draws the member in a specified restricted way.
```
We should actually fix this on all attributes...